Assets/Scripts/Actors/ActorPhysics.cs

```diff
@@ -343,7 +343,13 @@ public void AirMovement( Vector3 inputVec, bool forceUp = false )
 		}
 		else
 		{
-			_moveVec = inputVec * _jumpMoveSpeed;
+			_moveVec += inputVec * _jumpMoveSpeedChangeRate;
+			_moveVec.y = 0.0f;
+
+			if ( _moveVec.sqrMagnitude > _sprintMoveSpeed * _sprintMoveSpeed )
+			{
+				_moveVec = _moveVec.normalized * _sprintMoveSpeed;
+			}
```

Comment:
The horizontal move speed while in the air shouldn't be a constant value, that was the problem with your implementation. If the player is in the air and moving forward and starts holding the back button, they should slow down to a stop (horizontally) before starting to move backwards. So if we normalize the movement speed then there's no in-between, they always move at a set speed in whatever direction they're pressing.